lib/fetchers/local.rb

```diff
@@ -39,12 +47,39 @@ def self.resolve_from_string(target)
       target if File.exist?(target)
     end
 
-    def initialize(target)
+    def initialize(target, opts = {})
       @target = target
+      @do_vendor = opts[:do_vendor]
```

Comment:
Wait, we have a constructor option on whether to vendor, AND we have an effectively global variable (Inspec::BaseCLI.command) we use to make the same decision?

Reply:
Yup, great catch. This is only used in one place so we can remove the instance variable.